docs/intro.md

```diff
@@ -4,4 +4,16 @@ sidebar_position: 1
 
 # Intro
 
-A hacker's guide to hackathons, made by the HackUTD Team.
+Most hackathon projects will want some sort of user interface - after all, you want the judges to be able to visualize your idea, right? Many projects also contain some sort of system that deals with data, whether that be user data or data from external sources. This guide explains and provides resource to learn about user interfaces and data driven systems.
```

Comment:
lololol saying "after all, you want judges to visualize ur idea, right?" could come off lil condescending. Since this is the first introduction of top level concepts u can consider replacing this blurb w -> "Web apps can seem super scary and complicated, but after reading our guide, you can be confident to making one of your own! \n Let's think about YouTube. When the website loads in, all the videos on your homepage are presented super cleanly and you can click on any one of your choosing. We call this the **frontend** and all the data of what each video is we call the **backend**. Keep scrolling to learn more about each!" Sm like this (but prob shorter) so it's very example based and they can make connections to things they're familiar with.